changes.go

```diff
@@ -251,6 +252,74 @@ type CommentInput struct {
 	Message   string        `json:"message,omitempty"`
 }
 
+// RobotCommentInput entity contains information for creating an inline robot comment.
+// https://gerrit-review.googlesource.com/Documentation/rest-api-changes.html#robot-comment-input
+type RobotCommentInput struct {
+	CommentInput
+
+	// The ID of the robot that generated this comment.
+	RobotId string `json:"robot_id"`
+	// An ID of the run of the robot.
+	RobotRunId string `json:"robot_run_id"`
+	// URL to more information.
+	Url string `json:"url,omitempty"`
```

Comment:
This should be `URL` per https://golang.org/s/style#initialisms.

Reply:
Both will be fixed in #62